sims/simupop_timings.txt

```diff
@@ -0,0 +1,5 @@
+time	mem	arg	queue	N	rho	time_per_gen
+57039.67	161462324	True	False	10000.0	10000.0	0.5703967
+39301.47	156774712	True	False	10000.0	1000.0	0.3930147
+43964.38	4771404	True	False	1000.0	10000.0	4.396438
```

Comment:
@ashander noted that this one is slower by a factor of 10.  This seems very strange?